### ISSUE TITLE
Fix aria-required-children and html-has-lang accessibility violations on series listing page

### ISSUE THREAD

**Issue by vh8154** [pulibrary/repecwp#152]
### Steps to recreate
1. Go to [Repec series listing page](https://repec-prod.princeton.edu/)
2. Run Axe DevTools on this page

### URL
https://repec-prod.princeton.edu/

### Issues
- Certain ARIA roles must contain particular children 
- html element must have a lang attribute

### Severity
Critical

### WCAG Success Criterion
[WCAG 1.3.1 Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
[WCAG 3.1.1 Language of Page](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-doc-lang-id.html)

### WCAG Conformance level (A, AA, AAA)
A

### User impact
1. Certain ARIA roles not having particular children can cause confusion for assistive technologies such as screen readers to convey information accurately.
2. Screen readers use the lang attribute to determine the language of content and without it, users may hear the wrong pronunciations or intonations.

### Recommended fix
1. To solve this problem, you need to fix the following: Element has children which are not allowed: [role=search]
2. To solve this problem, you need to fix the following: The <html> element does not have a lang attribute

### Acceptance criteria
- [ ] When removing pending in the accessibility_spec for the series listing page, rspec tests run without error.
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...
